wandb/cli/cli.py

```diff
@@ -1243,6 +1243,12 @@ def launch_sweep(
     default=None,
     help="Path to the Dockerfile used to build the job, relative to the job's root",
 )
+@click.option(
+    "--template-variables",
+    "-T",
+    default=None,
+    help="JSON string containing values for template variables, only usable with the queue option. Template variables can also be passed via the `template_variables` key in the config",
+)
```

Comment:
I find our use of json strings are cli arguments kinda smelly, but maybe I am off base. I feel like the better workflow is to provide a file. I also find yaml easier to read/write for this kind of config stuff.

Reply:
Also I think we need to explain the expected structure here

Reply:
Dropped it

Reply:
Another (better?) options is to allow for something like:
-t var1=x -t var2=y

but this gets tough because the types are harder to infer so I went this route. We can allow for a file, but then it should just be the `template_variables` key in the config file. Happy to drop this option